modules/ocm-enabling-autoscaling-nodes.adoc

```diff
@@ -0,0 +1,21 @@
+
+// Module included in the following assemblies:
+//
+// nodes/nodes/nodes-enabling-autoscaling-nodes.adoc
+
+[id="ocm-enabling-autoscaling_{context}"]
+= Enabling autoscaling nodes in an existing cluster using OCM
+
+Enable autoscaling for worker nodes in the machine pool definition from the OCM console.
+
+.Procedure
+
+. From the link:https://console.redhat.com/[OCM console], navigate to the *Clusters* page and select the cluster for which you wish to enable autoscaling.
+
+. On the selected cluster, select the *Machine pools* tab.
+
+. Click the "More Options" at the end of the machine pool with autoscaling and select *Scale*.
```

Comment:
Is "More Options" actually shown on the page? If so, it should be bold/asterisks. If not, then it should be reworded another way